test/functional/normal/tab_spec.lua

```diff
@@ -0,0 +1,230 @@
+local helpers = require('test.functional.helpers')(after_each)
+
+local clear = helpers.clear
+local eq = helpers.eq
+local feed = helpers.feed
+local redir_exec = helpers.redir_exec
+
+describe('tabpage/earlier', function()
```

Comment:
let's put this in `/test/functional/autocmd/tabnewentered_spec.lua` rather than creating a new test file.

(I would like to combine all of the `/test/functional/autocmd/tab*_spec.lua` files as well, we don't need all of this granularity)

Reply:
The tests are now in `/test/functional/autocmd/tabnewentered_spec.lua`.